src/edu/csus/ecs/pc2/ui/ProblemsPane.java

```diff
@@ -142,10 +142,11 @@ private MCLB getProblemListBox() {
         if (problemListBox == null) {
             problemListBox = new MCLB();
 
-            Object[] cols = { "Problem Name", "# Test Cases", "Input Method", "Judging Type", "Short", "Time Limit(secs)", 
-                    "Max Output(KB)", "Input Validator", "I.V. Status", "Output Validator", "O.V. Command", "Groups" };
+            Object[] cols = { "Problem Name", "Test Cases", "Input Method", "Judging Type", "Short", "Time Limit(Secs)",
+                    "Max Output (KB)", "Mem Limit(MB)", "Input Validator", "I.V. Status", "Output Validator", "O.V. Command", "Groups" };
```

Comment:
I must admit it mystifies me why in the above line a blank was added preceding `(KB)`, but then `mem Limit(MB)` was added without a blank in front of the `(MB)`...

Reply:
Also, the "Secs" was capitalized in Time Limit.  I see no reason for the space, i looked at the pane with and without the space and it wasn't done for width reasons.  I removed the space.  Changed in [this commit.](https://github.com/pc2ccs/pc2v9/pull/732/commits/609d677d44262d1dbc6708fe0d923fc19db331d9)